src/main/kotlin/org/rust/ide/annotator/fixes/CompareZeroFix.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsCastExpr
+import org.rust.lang.core.psi.RsExpr
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.types.ty.TyNumeric
+import org.rust.lang.core.types.type
+
+class CompareZeroFix private constructor(expr: RsExpr) : LocalQuickFixAndIntentionActionOnPsiElement(expr) {
+    override fun getFamilyName() = "Compare with zero instead"
```

Comment:
I think `instead` is redundant in context of quick fix that supposed to modify code by default